api/include/vehicles/kia_soul_niro.h

```diff
@@ -0,0 +1,420 @@
+/**
+ * @file kia_soul_niro.h
+ * @brief Kia Soul Niro specific macros.
+ *
+ */
+
+
+#ifndef _KIA_SOUL_NIRO_PLATFORM_INFO_H_
+#define _KIA_SOUL_NIRO_PLATFORM_INFO_H_
+
+
+#include <stdint.h>
+
+
+// ********************************************************************
+//
+// WARNING
+//
+// The values listed here are carefully tested to ensure that the vehicle's
+// components are not actuated outside of the range of what they can handle.
+// By changing any of these values you risk attempting to actuate outside of the
+// vehicle's valid range. This can cause damage to the hardware and/or a
+// vehicle fault. Clearing this fault state requires additional tools.
+//
+// It is NOT recommended to modify any of these values without expert knowledge.
+//
+// ************************************************************************
+
+
+// ****************************************************************************
+// OBD MESSAGES
+// ****************************************************************************
+
+/*
+ * @brief ID of the Kia Soul's OBD steering wheel angle CAN frame.
+ *
+ */
+#define KIA_SOUL_OBD_STEERING_WHEEL_ANGLE_CAN_ID ( 0x2B0 )
+
+/*
+ * @brief ID of the Kia Soul's OBD wheel speed CAN frame.
+ *
+ */
+#define KIA_SOUL_OBD_WHEEL_SPEED_CAN_ID ( 0x4B0 )
```

Comment:
CAN IDs
wheel speed is 0x386
vehicle speed is 0x371

Reply:
9394cc0